_posts/data_reuse_000458.md

```diff
@@ -0,0 +1,43 @@
+# Unraveling Anesthesia's Effect on Brain Activity
+_Data Reuse Highlight: This blog entry is part of an ongoing series that showcases examples of data reuse and reanalysis using publicly available data on DANDI._
+
+Have you ever wondered what happens to your brain when you're under anesthesia? What neural changes coincide with that dramatic loss of consciousness? In this post, we'll explore how two different research teams investigated this fascinating question using the same data from DANDI Archive's Dandiset #000458.
+
+## The Original Study: Peering into the Anesthetized Brain
+In June 2023, Dr. Leslie Claar and colleagues from the Allen Institute published their groundbreaking findings in the journal _eLife_ in the article, “Cortico-thalamo-cortical interactions modulate electrically evoked EEG responses in mice”. Their approach was comprehensive: they stimulated mouse cortex while simultaneously recording with EEG and neuropixels probes, comparing brain activity during wakefulness and under isoflurane anesthesia.
+
+In awake mice, Claar's team observed a robust event-related potential (ERP) when they delivered pulsatile electrical stimulation to deep layers of the motor cortex. This ERP coincided with a distinct pattern of neural spiking, both locally in the motor cortex and in distant sensorimotor-related thalamic nuclei. The response followed a three-act play: initial excitation within 25 milliseconds, a period of quiescence for about 125 milliseconds, and finally, a strong rebound excitation before returning to baseline.
+When the mice were under anesthesia, the story changed dramatically. The ERP's amplitude and complexity plummeted in response to the same stimulation. While the initial excitation persisted in the motor cortex, it vanished in the thalamic nuclei. The rebound excitation, along with its corresponding ERP component at around 180 milliseconds, was significantly weakened.
+
+Based on these observations, Claar and colleagues proposed that interactions between the cortex and thalamus are responsible for the rebound excitation phase of the stimulus response. This rebound excitation, in turn, drives the second component of the ERP – a crucial signal of wakefulness.
+
+After their experiments, Claar and her team published their data on DANDI, ensuring that it would remain publicly accessible for any future researchers who wanted to use it.
```

Comment:
was it before or after article was published?

it is kinda important since if before article published  -- we can show this as an example of sharing before publication and ripping benefits from people discovering new stuff 'in addition' etc.  If shortly after -- also could be some story to tell ;-)

Reply:
The Dandiset and publication were developed and released pretty much simultaneously.

Dandiset was created 3/16/23 and last updated 6/16/23.
First version of the publication was released 2/9/23, and final version was published 6/26/23.

Reply:
ok, so sharing was delayed until a publication and initiated after an initial paper draft.  May be worth adding to the story something along the line of "Dataset preparation was done in concert with paper going through revisions to potentially satisfy curious paper reviewers, and helped to ensure that dataset includes all relevant to the article  data and metadata".